### DISCUSSION TITLE
Show/Hide per page selection

### DISCUSSION THREAD

**Discussion by jobad** [filamentphp/filament#2950] [Help]
hello, has anyone here have tried not showing the per page selection if there are no result found? Currently the pagination/index.blade.php core file does not have a condition to hide the per page selection it there are no records found. Thanks!

<img width="352" alt="Screen Shot 2022-06-28 at 2 59 56 PM" src="https://user-images.githubusercontent.com/137897/176114317-c6edae27-0984-472b-a78d-e95bfd59816c.png">

**Comment by ManissDev** [comment]


**Reply by danharrin** [reply]
Please don't override views, this can just be a PR to the core instead.

https://filamentphp.com/blog/publishing-views-in-laravel

**Reply by ManissDev** [reply]
Sure...

**Reply by danharrin** [reply]
This is now in the core, so you don't need to override anything
